apps/test/unit/templates/studioHomepages/BorderedCallToActionTest.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {expect} from '../../../util/reconfiguredChai';
+import {isolateComponent} from 'isolate-components';
+import {UnconnectedBorderedCallToAction} from '@cdo/apps/templates/studioHomepages/BorderedCallToAction';
+import sinon from 'sinon';
+import * as utils from '@cdo/apps/utils';
+
+describe('BorderedCallToAction', () => {
+  const headingText = 'Do Something';
+  const descriptionText = 'Get started now';
+  const buttonText = 'Get to it';
+  const buttonUrl = '/my/path';
+  const defaultProps = {
+    headingText,
+    descriptionText,
+    buttonText,
+    buttonUrl
+  };
+
+  describe('default behavior', () => {
+    const borderedCtA = isolateComponent(
+      <UnconnectedBorderedCallToAction {...defaultProps} />
+    );
+    it('renders a heading', () => {
```

Comment:
style nit: add empty newlines between `describe` and `it` statements for readability

Reply:
ah, sorry, i should've been more specific -- we should put empty newlines between individual `it` statements as well. i usually structure the skeleton of tests like this:

```javascript
describe('', () => {
  it('', () => {});

  it('', () => {});
});

describe('', () => {
  it('', () => {});

  it('', () => {});
});
```